bin/gen-browser-infos.js

```diff
@@ -0,0 +1,13 @@
+const caniuse = require('caniuse-api');
```

Comment:
how is this hooked in the build system?

Reply:
👍

Reply:
We can do this. I was not sure if we should add this 1.x MB lib even though we just need this simple piece of data.